.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "arrowParens": "always",
+  "tabWidth": 2,
+  "singleQuote": false,
+}
```

Please add a newline:
```suggestion
}

```